src/main/java/frc/robot/Robot.java

```diff
@@ -275,41 +275,12 @@ public void robotInit() {
                 .withTimeout(0.75));
     controller.rightBumper().whileTrue(swerve.stopWithXCmd());
 
-    // ---New climb----
-    operator
-        .rightBumper()
-        .toggleOnFalse( // TODO ???
-            Commands.parallel(
-                climber.extendClimb(),
-                leds.setBlinkingCmd(new Color("#ff0000"), new Color("#ffffff"), 15.0)
-                    .until(
-                        () ->
-                            climber.getPosition().getDegrees()
-                                > ClimberSubsystem.CLIMB_ANGLE.getDegrees())
-                    .andThen(
-                        leds.setBlinkingCmd(new Color("#00ff00"), new Color("#ffffff"), 15.0))));
-    controller
-        .x()
-        .onTrue(
-            climber
-                .retractClimb()
-                .alongWith(
-                    leds.setRainbowCmd())); // TODO not sure yet how the actual trap scoring works
-
-    // Heading reset
-    controller
-        .leftStick()
-        .and(controller.rightStick())
-        .onTrue(Commands.runOnce(() -> swerve.setYaw(new Rotation2d())));
-
     controller
         .y()
-        .and(() -> elevator.getExtensionMeters() > 0.9 * ElevatorSubsystem.CLIMB_EXTENSION_METERS)
+        .and(() -> climber.getRotations() > 0.9 * ClimberSubsystem.CLIMBER_MAX_ROTATIONS)
```

Comment:
shuold this be MAX_ROTATIONS or CLIMB_ROTATIONS?

Reply:
probably max since if its only at 90% of the climb rotations it won't climb right but idk